### DISCUSSION TITLE
Adding authors

### DISCUSSION THREAD

**Discussion by mattiasthalen** [olivierlacan/keep-a-changelog#523] [Q&A]
How would you recommend adding author(s)?
Some releases are made by me, some are made by others and a third is a combination of several authors.
I would really like to keep track of this in the changelog.

Should I just add an authors section and/or add the author for each change/addition/removal?
Example:
`
### Author
- author_1@example.com

### Added
- Pick released by
- Ship method [author_2@example.com]
- Pre-pack flag [author_3@example.com]

### Changed 
- Line date associoation.  [author_2@example.com]
- Changelog format.
- Start following semantic versioning properly.  [author_3@example.com]
`

**Comment by booch** [comment]
I feel like listing the author in each line of changes is sufficient. I don't think the **Author** section is usually very useful.

If seems like it'd be OK to have a paragraph at the top of the section for each version, with project-related announcements, calling out things like a new maintainer, or even thanking new authors joining in.
